proto/src/determined/trial/v1/trial.proto

```diff
@@ -320,3 +321,68 @@ message TrialSourceInfo {
   // Type for this trial_source_info
   TrialSourceInfoType trial_source_info_type = 5;
 }
+
+// Flat run respresentation.
```

Comment:
can we make fields as required https://github.com/determined-ai/determined/blob/314d44f950c1d1a173f27be576c687789047873c/proto/src/determined/trial/v1/trial.proto#L89/

so our binding generation doesn't make them optional

Reply:
Done in `run.proto`